.ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -66,6 +75,52 @@ wait_for_reboot() {
 	done
 }
 
+wait_mcp_update() {
+	local delta="${1:-900}"
+	local sleep_time=60
+	# The machineconfigpool is fine when all the workers updated and are ready,
+	# and none are degraded.
+	local ready_count=0
+	local degraded_count=0
+	local machine_count=$(oc get mcp worker -o jsonpath='{.status.machineCount}')
+
+	if [[ -z "$machine_count" && "$machine_count" -lt 1 ]]; then
+		warn "Unabled to obtain the machine count"
+		return 1
+	fi
+
+	echo "Set timeout to $delta seconds"
+	local timer_start=$(date +%s)
+	# The ready count might not have changed yet, so wait a little.
+	while [[ "$ready_count" != "$machine_count" && \
+		"$degraded_count" == 0 ]]; do
+		# Let's check it hit the timeout (or not).
+		local now=$(date +%s)
+		if [ $(($timer_start + $delta)) -lt $now ]; then
```

Comment:
I guess you wouldn't really have to run this addition every time but it doesn't matter a ton

Reply:
True. I will change it.

Reply:
Done.